src/app/info/WikiItems.tsx

```diff
@@ -94,13 +115,98 @@ const WikiItems: React.FC<WikiItemsProps> = ({ rows }) => {
         const indexToRemove: number = displayRows.findIndex(
             (sortedRow) => row.wiki_key === sortedRow.wiki_key
         );
-        setDisplayRows((DisplayRows) => {
-            const temp = DisplayRows.slice();
+        setDisplayRows((displayRows) => {
+            const temp = displayRows.slice();
             temp.splice(indexToRemove, 1);
             return temp;
         });
         return indexToRemove;
     };
+
+    const [isSwapMode, setIsSwapMode] = useState(false);
+
+    const swapRows = async (row1: DbWikiRow, upwards: boolean): Promise<void> => {
```

Comment:
can we change this so instead of a boolean we have a direction, and the since it's a funciton that is called on a single row that the name reflects that?
`const moveRow = async (row: DbWikiRow, direction: "up" | "down"): Promise<void> => {`
It'll make the intention a lot clearer where it's called